use-shopping-cart/src/util.js

```diff
@@ -2,13 +2,16 @@ import { useStorageReducer } from 'react-storage-hooks'
 
 export const isClient = typeof window === 'object'
 
-export const formatCurrencyString = ({ value, currency, language }) => {
```

Comment:
We still want a language option. I believe we just need to use `isClient` when we set the default value. Which we should probably do up here rather than in the function of the body so that it's obvious that it has a fallback so that no one tries to replicate this logic.

Reply:
Something like:

```js
export const formatCurrencyString = ({ value, currency, language = isClient ? navigator.language : 'en-US' }) => {
  const numberFormat = new Intl.NumberFormat(language, {
    style: 'currency',
    currency,
    currencyDisplay: 'symbol'
  })
```

Reply:
Cool, going to do this and will resolve